omod/src/main/java/org/openmrs/module/appointmentschedulingui/page/controller/AppointmentTypePageController.java

```diff
@@ -23,6 +23,9 @@ public void get(PageModel model,
 
         if(appointmentTypeId!=null){
```

Comment:
thanks @The-Lady  why not check for the null id just before this line  ie  if(appointmentTypeId ==null){ 

 }

Reply:
Appointment Type can be null even when Appointment Type ID is specified in the URL and is not null. This is because there has been no appointment type created for that appointment type id. So after appointment type id (provided in the URL) has been checked for not null, it has to be verified that there is actually an appointment type mapped to it before displaying the page and/or proceeding further as after this method appointment type is used for reference in the codebase and not appointment type id.